lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -16,6 +17,8 @@ <h3 class="error-msgs" id="error-msg"></h3>
             <span class="code_tip">${_('Click to generate Registration Codes')}
             <a id="registration_code_generation_link" href="#reg_code_generation_modal" class="add blue-button">${_('Generate Registration Codes')}</a>
             </span>
+            <span class="tip">${_("Click to make Transactions'")} <a id="make_invoice_transaction_link" href="#make_invoice_transaction_modal" rel="leanModal"
+                   class="add blue-button">+ ${_("Make Transaction")}</a></span>
```

Comment:
"Make transactions" sounds a little strange.  Maybe "Click to add invoice credits and refunds"